app/models/sent_scheduled_message.rb

```diff
@@ -9,4 +9,20 @@ class SentScheduledMessage < ActiveRecord::Base
   validates :sent_on, presence: true
 
   delegate :slack_username, to: :employee
+
+  def self.filter(params)
+  	@employee = Employee.find_by slack_username: params[:slack_username]
+
+    results = self.all.where("lower(message_body) like ?", "%#{params[:message_body].downcase}%")
+
+    if @employee
```

Comment:
what happens if you do not use `return`? As I am reading it, I would expect both to return the same